src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -269,8 +269,7 @@ await WriteJsonAsync(new DynamicJsonValue
                             $"{_subscriptionConnectionsState.GetConnectionsAsString()} is released");
                     }
 
-                    var timeout = TimeSpan.FromMilliseconds(Math.Max(250, (long)_options.TimeToWaitBeforeConnectionRetry.TotalMilliseconds / 2) + random.Next(15, 50));
-                    await Task.Delay(timeout);
+                    await Task.Delay(random.Next(250, 2500));
```

Comment:
1. We should pass here cts?
2. Prev logic was to send it +/- every 2500ms (assuming default value for TimeToWaitBeforeConnectionRetry), now you are sending it between 250 and 2500ms.
```csharp
await Task.Delay(random.Next(2400, 2600), CancellationTokenSource.Token);
```

Reply:
done